README.md

```diff
@@ -19,18 +19,30 @@ The extension provides a single module - **MONAIViz** - which currently supports
 
 ## Installing Plugin
 
-### Prerequisites
-You need to install MONAI and dependencies before using this plugin.
+### Installing 3D Slicer
 
-- Install the [PyTorch extension](https://github.com/fepegar/SlicerPyTorch)
-- Open Python Console (View Menu) in 3D Slicer
-- Run the following command:
-  - `pip_install("monai[itk,nibabel]")`
-- Restart 3D Slicer
+To use MONAIViz with 3D Slicer, you'll need to download and install 3D Slicer. MONAIViz supports the stable and preview versions of 3D Slicer, version 5.3 or higher. For more information on installing 3D Slicer, check out the [3D Slicer Documentation](https://slicer.readthedocs.io/en/latest/user_guide/getting_started.html#installing-3d-slicer)
```

Comment:
Since we will be releasing Slicer 5.4 this week, we could probably change this to `version 5.4 or higher`